### ISSUE TITLE
Add ENB Dropdown to allow users to customize ENB

### ISSUE THREAD

**Issue by TaranRoth** [UltimateSkyrim/ultimate-skyrim-launcher#215]
Options would be Stock ENB, Custom ENB, No ENB
Stock ENB would check if ENB files are present on launch, and if not, copy them over
Custom ENB wouldn't touch ENB files at all (if a user selects this it would be assumed they know what they're doing)
No ENB would scan for any ENB files present on launch and delete them if they are

**Comment by MattLish** [comment]
No longer needed once #296 is done
